src/main/scala/is/hail/rvd/OrderedRVD.scala

```diff
@@ -354,28 +365,36 @@ class OrderedRVD(
     spec.write(sparkContext.hadoopConfiguration, path)
     partitionCounts
   }
-}
 
-object OrderedRVD {
-  type CoercionMethod = Int
+  def zipPartitionsPreservesPartitioning[T: ClassTag](
+    newTyp: OrderedRVDType, that: RDD[T]
+  )(zipper: (Iterator[RegionValue], Iterator[T]) => Iterator[RegionValue]
```

Comment:
Should zipper be on its own line?

Reply:
No opinion just asking.

Reply:
IIRC, the full signature is >80 characters, so I went for the @patrick-schultz style function definition (with each arg on it's own line), but had to improvise a bit since there are two argument lists.

Reply:
Oh! Yes, I think `that` should also have its own line.